assets/scripts/streets/StreetMetaData.jsx

```diff
@@ -101,6 +102,19 @@ class StreetMetaData extends React.Component {
     return text
   }
 
+  renderGeotag = (readOnly, street) => {
```

Comment:
Nitpicky, but I like ordering arguments in order of importance. `street` is more relevant for the function than `readOnly` is.